python/triton/ops/matmul_perf_model.py

```diff
@@ -0,0 +1,133 @@
+import torch
+import triton
+import triton._C.libtriton.triton as _triton
+import heapq
+
+def get_dram_bw(backend=None, device=None):
+  ''' return DRAM bandwidth in GB/s '''
+  # assert backend == CUDA
+  if not backend:
+    backend = _triton.runtime.backend.CUDA
+  if not device:
+    device = torch.cuda.current_device()
+  mem_clock_khz = _triton.runtime.memory_clock_rate(backend, device)
+  bus_width = _triton.runtime.global_memory_bus_width(backend, device)
+  bw = mem_clock_khz * bus_width * 2 // 1024 // 1024 // 8 # In GB/s
+  return bw
+
+def get_matmul_tput(backend, device, num_ctas, num_warps):
```

Comment:
Same comment  as `get_dram_bw`. Also, it seems like this just returns the maximum tflops of the device, which is unrelated to matmul, so maybe `get_compute_tflops` would be a better name?

Reply:
I'm thinking about this. How about we rename this as `get_tensorcore_tflops`?

Reply:
Yep that's even better :)